out/_next/static/chunks/pages/_error-9b7125ad1a1e68fa.js

```
@@ -0,0 +1 @@
+(self.webpackChunk_N_E=self.webpackChunk_N_E||[]).push([[731],{6194:(_,n,e)=>{(window.__NEXT_P=window.__NEXT_P||[]).push(["/_error",function(){return e(6218)}])}},_=>{var n=n=>_(_.s=n);_.O(0,[636,593,792],()=>n(6194)),_N_E=_.O()}]);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)